### ISSUE TITLE
1.8.3 fails to connect to Pulseaudio; 0.7.2 does not

### ISSUE THREAD

**Issue by keantoken** [DeaDBeeF-Player/deadbeef#2371]
This started happening after a massive apt update. I think I just updated to Debian 10. I reset the computer and the issue persists.

If I watch pulseaudio -vvvv while trying to play a track in 1.8.3, there is no change in the log, as if deadbeef is simply failing to reach pulseaudio. If I open version 0.7.2, it is able to play fine.

Before the update I was able to play music with 1.8.3.

**Comment by keantoken** [comment]
The terminal gives this message:

pa_simple_new failed: Connection refused

**Comment by keantoken** [comment]
Okay, I opened the Pulseaudio plugin config from the Plugins tab. In the output field there was the test "default" which is the name of my pulseaudio soundcard. I think this was carried over in the config file from a previous version of Deadbeef. The window says to leave blank for default, so I cleared the field and now it works.

I was only just barely clever enough to figure that out. Not sure why it failed because "default" should be the correct card to use.
